### PR TITLE
this solves issue #1351

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
@daren-thomas please check, thanks!

### CODE REVIEW

**Comment by JIMENOFONSECA** [comment]
@shanshanhsieh would you mind taking a look at this small patch? thanks!

**Comment by shanshanhsieh** [comment]
sure @JIMENOFONSECA , but what should be checked?
the link is working but it seems like the cea test is still failing.

**Comment by shanshanhsieh** [comment]
update, cea test of this branch works on my laptop

File: docs/getting-started.rst [hunk 1]
```diff
@@ -22,6 +22,12 @@ Install and set up
 :doc:`installation-on-euler`.
 
 
+Download a test case study
+--------------------------
+
+Just go to this `link <https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/master/cea/examples>`__.and unzip the case study where necessary.
```
Comment (daren-thomas) [inline docs/getting-started.rst]: you know... you could also just do `cea extract-reference-case --destination ~/my/favorite/folder`...

**Pull request merged**